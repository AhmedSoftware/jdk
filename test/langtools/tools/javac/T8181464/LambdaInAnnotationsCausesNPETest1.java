/* @test /nodynamiccopyright/
 * @bug 8181464
 * @summary Invalid lambda in annotation causes NPE in Lint.augment
 * @modules java.compiler
 *          jdk.compiler
 * @compile Anno.java AnnoProcessor.java
 * @compile -J-XX:+UnlockExperimentalVMOptions -J-XX:hashCode=2 Anno.java AnnoProcessor.java
 * @compile/fail/ref=LambdaInAnnotationsCausesNPETest1.out -XDrawDiagnostics -processor AnnoProcessor -proc:only LambdaInAnnotationsCausesNPETest1.java
 */

@Anno(value = x -> x)
class LambdaInAnnotationsCausesNPETest1 {}
