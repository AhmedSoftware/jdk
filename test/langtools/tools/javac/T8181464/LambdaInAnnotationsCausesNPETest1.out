LambdaInAnnotationsCausesNPETest1.java:11:15: compiler.err.expression.not.allowable.as.annotation.value
1 error
